cadc-download-manager/build.gradle

```diff
@@ -24,13 +24,12 @@ dependencies {
     implementation 'org.opencadc:cadc-util:[1.6,)'
     implementation 'org.opencadc:cadc-registry:[1.1.1,)'
     implementation 'org.opencadc:cadc-app-kit:[1.0,)'
-    implementation 'org.opencadc:cadc-vos:[1.0,)'
+    implementation 'org.opencadc:cadc-vos:1.2.3'
```

Comment:
Pinned on purpose?

Reply:
Yes, due to the JDK 1.8 requirement.

Reply:
I suggest you add a comment so we remember.